OpenApiSpecs/exchangeSetService_OpenApi_definition_internal.yaml

```diff
@@ -84,6 +87,72 @@ paths:
       callbacks:
         fulfilmentResponse:
           $ref: "#/components/callbacks/fulfilmentResponse"
+     
+  /v2/exchangeSet/s100/productNames:
+    post:
+      summary: Provide all the latest releasable baseline data for a specified set of Products.
+
+      operationId: postS100ProductNames
+
+      description: |
+        Given a list of Product names, return all the products that are releasable.
+ 
+        ## Business Rules:
+ 
+        Only Products that are releasable at the date of the request will be returned.
+
+        If valid Products are requested then Product exchange set with baseline data including requested Products will be returned.
+        
+        If a requested Product has been cancelled or replaced, then the replacement Product will not be included in the response payload. Only the specific Products requested will be returned.
+
+        If none of the Products requested exist then exchange set with baseline releasable data without requested Products will be returned
+        
+      tags:
+         - s100
+         
+      parameters:
+        - $ref: "#/components/parameters/callbackUri"
+        
+      requestBody:
+        description: |
+          The JSON body containing product names.
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/productNames"
+            example: ["101GB40079ABCDEFG", "102NO32904820801012", "104US00_CHES_TYPE1_20210630_0600", "111US00_ches_dcf8_20190703T00Z"]
+
+      responses:
+        "200":
+          description: A JSON body that indicates the URL that the Exchange Set will be available on as well as the number of products in that Exchange Set.
```

Comment:
Shall we make it 202 - Accepted in v2 apis as exchange set generation is asyncrounous process? We already have a "status" url in the response.

Reply:
Updated status code to 202.